dbcon/mysql/ha_mcs_ddl.cpp

```diff
@@ -981,7 +980,9 @@ int ProcessDDLStatement(string& ddlStatement, string& schema, const string& tabl
 
                     if (compressionType == 1) compressionType = 2;
```

Comment:
Can we use not magic numbers here and below in all similar places, but enum values, some function or something like that?

Reply:
We can set a compression type by session variable "columnstore_compression_type=1" or "2" or whatever we have.

Current distribution of compression types:
`
SNAPPY = 0,
SNAPPY = 1,
SNAPPY = 2,
LZ4 = 4,
`
Not sure how to make an enum from this.
Do you mean like
`
SNAPPY0 = 0
SNAPPY1 = 1
SNAPPY2 =2
`

Reply:
I mean a function that maps old bad int compression type to the enum value, for example. Or, if it's possible, do that mapping while setting/getting variable.

Reply:
Sorry If I didn't get it correctly, do you mean to change the `int type` compression type everywhere in source to `enum type`? 
If, so, may be we can separate this to another pull request, because the change could be a huge, the `int type` for compression is using in a lot of place in some structs, classes and so on.
BTW, I totally agree that a lot of places in CS should be rewritten, my opinion that it would be we better if we can separate this work from "feature" patches, but I might be wrong about it.
Thanks.

Reply:
I told Alexey to address this. However you are right and this looks like a big patch that is outside the scope.